source/Octopus.Client.Tests/Integration/Repository/MachineRepositoryTest.cs

```diff
@@ -1,16 +1,21 @@
-using System;
-using System.IO;
-using System.Reflection;
+using System.Collections.Concurrent;
+using System.Linq;
 using Nancy;
+using Nancy.Extensions;
+using Newtonsoft.Json;
 using NUnit.Framework;
 using Octopus.Client.Extensibility;
 using Octopus.Client.Model;
+using Octopus.Client.Model.Endpoints;
 using Octopus.Client.Repositories.Async;
+using Octopus.Client.Serialization;
 
 namespace Octopus.Client.Tests.Integration.Repository
 {
     public class MachineRepositoryTest : HttpIntegrationTestBase
     {
+        private static ConcurrentDictionary<string, MachineResource> machines = new ConcurrentDictionary<string, MachineResource>();
```

Comment:
This isn't nice, but hopefully whoever uses this in the future notices that it's `Concurrent` for a reason.